src/Pages/Expenses.cshtml.cs

```diff
@@ -61,14 +62,17 @@ public class ExpensesModel : PageModel
         public bool ShowNext => CurrentPage < TotalPages;
         private UserDatabase UserDatabase { get; }
         private CurrencyConverter CurrencyConverter { get; }
+        public decimal CurrencyRate { get; set; }
         public string CurrencySymbol { get; private set; }
-        public ExpensesModel(EntryDatabase entryDatabase, ILogger<ExpensesModel> logger, IConfiguration configuration, UserDatabase userDatabase, CurrencyConverter currencyConverter)
+        private IMemoryCache Cache { get; }
+        public ExpensesModel(EntryDatabase entryDatabase, ILogger<ExpensesModel> logger, IConfiguration configuration, UserDatabase userDatabase, CurrencyConverter currencyConverter, IMemoryCache cache)
```

Comment:
Same with this line, well maybe the ones that are too long in page controllers I'll refactor when I'll go through them.